[Queues] fix omissions in get-started.md (#7513)
cloudflare/cloudflare-docs: content/queues/get-started.md

  https://<YOUR-WORKER-NAME>.<YOUR-ACCOUNT>.workers.dev
```

> Now make a POST request to your worker using cURL:

Reviewer: ```suggestion
Now make a POST request to your producer Worker using cURL:
```

> Now make a POST request to your producer Worker using cURL:

Reviewer: ```suggestion
After you have updated your `index.ts` file, make a POST request to your producer Worker using cURL:
```

```toml
[[queues.consumers]]

>  queue = "YOUR_QUEUE_NAME"

Reviewer: ```suggestion
 queue = "<YOUR_QUEUE_NAME>"
```

Adding <> back to indicate that this is a field the user needs to update.
